### PR TITLE
a9 parallel blob response

### PR DESCRIPTION
**Pull request by borisjaskerovich**

### CODE REVIEW

**Comment by ghost** [comment]
Build Passed

**Comment by ghost** [comment]
Build Passed

File: modules/a9ParallelBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  code: BIDDER_CODE,
+  aliases: ['a9Parallel'], // short code
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return true;
```
Comment (shlomisut) [inline modules/a9ParallelBidAdapter.js]: Maybe using a 'false' here would do the trick?

**Comment by ghost** [comment]
Build Passed

**Review by shlomisut — APPROVED** [review]

**Pull request merged**